config/redpanda-cluster.yaml

```diff
@@ -9,9 +9,10 @@ kafka:
     # username: redpanda
```

Comment:
why rename

Reply:
To make it more clear why this config exists.

Reply:
```suggestion
    # this file is used to simulate the security model on a cluster which uses OAuth for security
    # username: redpanda
```